### DISCUSSION TITLE
Smart lists?

### DISCUSSION THREAD

**Discussion by indy1725** [Athou/commafeed#1117] [Ideas]
I'd love to be able to set up a Smart list that takes all the current items and only shows me Today's news, rather than stuff that's piled up over the last few days. Or news that's sorted by a custom tag or keyword.

**Comment by VlaK0r** [comment]
It can already be done

**Reply by indy1725** [reply]
How? I don't see any way to set up anything other than adding new subscriptions and categories.

**Reply by VlaK0r** [reply]
![image](https://github.com/Athou/commafeed/assets/77666575/1bfc52ea-ffd1-44b1-9684-06726e498745)
We generate a new feed url from everything, add it and apply a filter to it.
Isn't that what you need?

**Reply by indy1725** [reply]
Hm, okay, I'll try that. It might work to filter for specific words, but it doesn't look like the pubDate tag is filterable.
